### DISCUSSION TITLE
Guild Festival

### DISCUSSION THREAD

**Discussion by Nubwos** [fedorovvl/tso_client#414] [General]
Hin

On the German server we play the guild festival. This is not possible with the client: Quests cannot be clicked, progress is not displayed (icon is not available), client often crashes...

LG

**Comment by Cdrwbw** [comment]
Don't use client to play the event. Don(t validate quest or message of coop aventure vith client, return on unity. Festival guild is not implemented in client.

**Comment by Stribrnak1** [comment]
Plánuje se implementace Festivalové guildy do klienta? Pokud ano, kdy se na to můžeme těšit?
Děkuji za odpověď, Stříbrňák

**Reply by Cdrwbw** [reply]
ne, žádná implementace se neplánuje, hrajte s klientem bez ověřování úkolů a kooperativních dobrodružství obsahujících vent předměty. Přepněte na untiy pro tyto účely.
přeložil deepl

**Reply by sroca-81** [reply]
if I want to try this implementation, where can I find documentation and/or api for swf and tso?

**Reply by GregSam** [reply]
all we got is this repository, all work is done on decompiled swf
